contracts/src/StakeTable.sol

```diff
@@ -323,7 +343,8 @@ contract StakeTable is AbstractStakeTable {
         // currentEpoch() + 1 (the start of the next full epoch), but in periods of high churn the
         // queue may fill up and it may be later. If the queue is so full that the wait time exceeds
         // the caller's desired maximum wait, abort.
-        (uint64 registerEpoch, uint64 queueSize) = this.nextRegistrationEpoch();
+        (uint64 registerEpoch, uint64 queueSize) =
+            this.nextAvailableEpoch(firstAvailableRegistrationEpoch, _numPendingRegistrations);
```

Comment:
There is the comment above how `_numPendingRegistrations` is not the total queue size.

https://github.com/EspressoSystems/espresso-sequencer/blob/072806816ed349315e47f695c689d55ea3537d5d/contracts/src/StakeTable.sol#L89-L91

But later we set it as such, although there is also a TODO in that comment

https://github.com/EspressoSystems/espresso-sequencer/blob/072806816ed349315e47f695c689d55ea3537d5d/contracts/src/StakeTable.sol#L213-L219

I think it would be nice if we named things a bit more explicitly. Since there are many epochs, queue sizes, etc. especially for those variables it's useful to be explicit.